plugins/vpp/model/acl/keys_agent_acl.go

```diff
@@ -14,7 +14,20 @@
 
 package acl
 
-const aclPrefix = "vpp/config/v1/acl/"
+import "github.com/ligato/vpp-agent/plugins/vpp/model"
+
+const (
+	// DB key prefix
+	aclPrefix = "vpp/config" + model.ProtoApiVersion + "acl/"
```

Comment:
This now seems less readable and might not be immediately obvious. Also it's so simple to just copy paste the key prefix.

Reply:
This is something we need to discuss more. There was an idea to already use the api version for something (currently it's always v1 and it hadn't changed since the beginning). 
Anyway, let's say you want to change the version number in every key, URL, just everywhere.  It will be easy to forgot something.